src/TropicalGeometry/TropicalPolyhedron/properties.jl

```diff
@@ -0,0 +1,308 @@
+@doc raw"""
+    vertices([as::Type{T} = PointVector,] P::TropicalPolyhedron)
+
+Returns an iterator over the vertices of `P`, that is, a minimal generating set `V`
```

Comment:
```suggestion
Return an iterator over the vertices of `P`, that is, a minimal generating set `V`
```

The convention is not to use third person singular (don't ask me why).